### DISCUSSION TITLE
issues disabled

### DISCUSSION THREAD

**Discussion by cbenard** [desjarlais/Scintilla.NET#13] [General]
Why are issues disabled @VPKSoft? Can they be enabled to track requested work?

I want to ask about forking https://github.com/Stumpii/ScintillaNET-FindReplaceDialog since the nuget packages that exist rely on the dead jacobslusser repo.

**Comment by VPKSoft** [comment]
Now the issues are enabled - seems to be a default setting in a forked repo 🤔

**Reply by cbenard** [reply]
Thanks. I opened #14.

**Comment by desjarlais** [comment]
Issues is enabled and will continue to be enabled.
